docs/examples.md

```diff
@@ -0,0 +1,363 @@
+                           :props {:first-name "Neo"
+                                   :last-name "Anderson"}})
+~~~
+
+### Find nodes in Neo4j
+
+The entry given to `find-nodes!` is a lookup representation. To learn more about the Clojure lookup representation see our [representations](representations.md) page
+
+~~~clojure
+(client/find-nodes! conn {:ref-id "p"
+                          :labels [:person]
+                          :props {:first-name "Neo"
+                                  :last-name "Anderson"}})
+~~~
+
+### Update and Delete Node
+
+See the section [Update and Delete Entity](#update-and-delete-neo4j-entity)
+
+
+## Relationship CRUD
+
+This section shows how to do basic CRUD operations on relationships through Neo4clj convenience functions.
+To learn more about the Clojure representation of a relationship please see our [representations](representations.md) page
+
+### Create a Relationship
+
+The keys `from` and `to` are Lookup representations. To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+~~~clojure
+(client/create-relationship! conn {:ref-id "p"
+                                   :type :employee
+                                   :from {:labels [:person] :props {:first-name "Neo"}}
+                                   :to {:id 12}
+                                   :props {:position "Developer"}})
+~~~
+
+### Find relationships in Neo4j
+
+To fetch relationships from the Neo4j database, please refer to the section [Get Graph](#get-graph)
+
+### Update and Delete Relationship
+
+See the section [Update and Delete Entity](#update-and-delete-neo4j-entity)
+
+
+## Update and Delete Neo4j Entity
+
+In this section you can find examples on how to update specific parts a Node or Relationship and how to delete them.
+
+### Update Node (Labels)
+
+To change the labels of a node we have added two convenience functions.
+Both the `add-labels!` and `remove-labels!` functions takes a lookup representation as second argument.
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Add labels
+
+~~~clojure
+(client/add-labels! conn {:id 45} [:person :salesman])
+~~~
+
+#### Remove labels
+
+~~~clojure
+(client/remove-labels! conn {:labels [:person] :props {:first-name "Neo"}} [:person :salesman])
+~~~
+
+### Update Node or Relationship (properties)
+
+To change the properties of a node or relationship we have added two convenience functions.
+Both the `update-properties!` and `replace-properties!` functions takes a lookup representation as second argument.
+
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Update properties
+
+This function will update an existing properties map based on the following rules.
+
+⋅⋅* Keys existing only in the given property map is added to the object
```

Comment:
This looks weird when markdown is rendered. Maybe delete the `..`?

Also it isn't clear what "the given property maps" is?

I'm assuming that it is the third argument? If the lookup representation gets documented in the text above:
> ... takes a lookup representation as second argument.

Why not do the same for "the given property map"?


I don't know if the following would be correct?:
> To change the properties of a node or relationship we have added two convenience functions.
Both will take a [lookup representation](representations.md) as second argument and a map with the properties that should be updated as the third.

Reply:
I had used MD for unordered sublist instead of just unordered list, fixed now. I have also added additional description to the function.